src/domain/resource/utils.js

```diff
@@ -92,7 +92,7 @@ export const getPrice = (minPriceString, maxPriceString, priceType, t) => {
     week: t('common.unit.time.week'),
   });
 
-  if (minPrice && maxPrice && minPrice !== maxPrice) {
+  if (minPrice >= 0 && maxPrice && minPrice !== maxPrice) {
```

Comment:
I think this will now include also `minPrice` `null`, which is probably not what we want.

Reply:
Customer wanted `null minPrice` to be rendered as 0. So, for now we want to include `null` value too.

Reply:
Ah ok. IMHO it is not really nice, as those fields are nullable for a reason, and that is to distinguish between "zero" and "not set". But of course there can be some practical reasons which I'm not aware of, and the customer is the king anyway, so all good here :)